src/redux/books/books.js

```diff
@@ -1,3 +1,6 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
 const defaultState = [
```

Comment:
- [ ] As per project requirements, We have to set the initial state to be an empty array of books. Kindly try to set the initial state to an empty array.

Reply:
* Done 👍